Bio/SeqIO/FastaIO.py

```diff
@@ -58,15 +58,16 @@ def SimpleFastaParser(handle):
     # (and any embedded \r which are possible in mangled files
     # when not opened in universal read lines mode)
     lines = []
+    ordinal_mapping = {32: None, 13: None}  # ord(" "), ord("\r")
     for line in handle:
         if line[0] == ">":
-            yield title, "".join(lines).replace(" ", "").replace("\r", "")
+            yield title, str.translate("".join(lines), ordinal_mapping)
```

Comment:
The Fasta parser could be much faster if we read in a block of data instead of reading line-by-line (which requires looking for the end-of-line marker every time), and then using `str.translate`.

Reply:
True, but this change seems worthwhile in and of itself?

Not sure if the original Pearson tools would also silently ignore tabs, but if so would adding tab to the ordinal_mapping make any noticeable difference in runtime?

Reply:
I don't know; as far as I know, we don't have timings for this change.